.travis.yml

```diff
@@ -42,9 +42,6 @@ env:
   - REQUIREMENTS=lowest
   - REQUIREMENTS=release
   - REQUIREMENTS=devel
-  - REQUIREMENTS=lowest CLEANCSS=3.4.19
```

Comment:
IMHO no, if we leave it we are not testing functionality but compatibility, the functionality cleaning the css is the same on both v3 & v4. Otherwise we should test for the current versions of ``requirejs``, ``node-scss``, ``jsmin`` etc